src/components/SiteHeader/index.js

```diff
@@ -271,7 +283,7 @@ const MenuItems = ({ onLogOut, showItems, onLinkClick }) => {
           >
             <StyledMenuButton
               iconSize={16}
-              iconLeft={<LogoutIcon focusable="false" />}
+              iconLeft={<LogoutIcon focusable="false"/>}
```

Comment:
I know this code was already there, but it might be an upgrade to make focusable false by default.

Reply:
Euh, I forgot about this remark. Let me make a note on the next story about the dashboard.